prime-router/src/main/kotlin/Sender.kt

```diff
@@ -41,6 +44,34 @@ open class Sender(
         }
     }
 
+    /**
+     * @property testManufacturer Sender a test manufacturer (Abbott, Roche, Quidel)
+     * @property dataAggregator Sender is a data aggregator (LifePoint, ImageMover, SimpleReport, Prescryptyve)
+     * @property facility Sender is a facility (CSV-uploader, urgent care, nursing homes, A1 Health--home health and hospice)
+     * @property hospitalSystem Sender is a hospital or large hospital system (HCA)
+     */
+    enum class SenderType {
```

Comment:
Do proper kdoc.  A description of the class, then comments on top of each attribute if needed.

Reply:
done